guide/how-it-works/wallet-privacy.md

```diff
@@ -0,0 +1,198 @@
+---
+layout: guide
+title: Wallet privacy
+description: An overview of how to help users maintain their financial privacy while using a bitcoin daily spending wallet.
+nav_order: 9
+parent: How it works
+permalink: /guide/how-it-works/wallet-privacy/
+main_classes: -no-top-padding
+image: /assets/images/guide/how-it-works/wallet-privacy/wallet-privacy-preview.jpg
+---
+
+{% include picture.html
+    image = "/assets/images/guide/how-it-works/wallet-privacy/wallet-privacy.jpg"
+    retina = "/assets/images/guide/how-it-works/wallet-privacy/wallet-privacy@2x.jpg"
+    mobile = "/assets/images/guide/how-it-works/wallet-privacy/wallet-privacy-mobile.jpg"
+    mobileRetina = "/assets/images/how-it-works/wallet-privacy/wallet-privacy-mobile@2x.jpg"
+    alt-text = "Illustration of an eye, with a cross on top of an ink blot background"
+    width = 1600
+    height = 600
+    layout = "full-width"
+%}
+
+<!--
+
+This page provides a general overview of privacy issues in bitcoin wallets.
+
+-->
+
+# Wallet privacy
+{: .no_toc }
+
+---
+
+<div class="glossary-toc" markdown="1">
+ * Table of contents
+{:toc}
+</div>
+
+---
+
+Privacy is about the ability to control what information is available to others, and what they can do with it. On this page, we will provide a soft introduction to the topic, with a focus on what product builders and users can do to improve their payment privacy.
```

Comment:
Agree

Reply:
Do you mean in the context of this page or in general?

Don't privacy policies usually also include descriptions on how data is used? Here's a screenshot from an App Store listing whose privacy section describes both what data is collected and how it is used ([App store guidelines](https://developer.apple.com/app-store/app-privacy-details/#data-type-usage)):
![app-store-app-privacy](https://user-images.githubusercontent.com/695901/186353470-6b58b79e-07b4-4efc-9f95-b3ff07598fc3.jpg)

Reply:
When accepting that policy though you are choosing to reveal it like Stephen said, once they have it they can do what they want with it

Reply:
> Do you mean in the context of this page or in general?

Both.

In general: consulted with 3 dictionaries and I can't find any mention of privacy meaning "to control what other people do with your information". The word typically refers to a state of concealment or the right to be in a state of concealment.

In the context of this page: privacy policies are a non-sequitur. They are not applicable because we are talking mostly about real-world privacy and bitcoin network privacy, not information we share with companies. But even if we do want to talk about company privacy policies, I'd argue that the privacy policy does not give us any control. A company can have a very strict sounding privacy policy and still have the ability to abuse the policy. If nobody figures out, the company gets away with it. If somebody does figure out, there may be legal action, but that doesn't change the fact that the abuse _already happened_.

In the screenshot you shared, that screen does not give the user any control over what TikTok does with their data. It only gives the user control over what data they choose to share.

I'm reading through both of these privacy pages and I'm having trouble finding an example of a place where it discusses revealing information to somebody and then having an ability to control what that person does with the information.

Reply:
Sure, let's go with your recommendation.